recipes/libcurl/all/test_package/conanfile.py

```diff
@@ -21,7 +21,7 @@ def test(self):
 
         if "arm" in self.settings.arch:
             self.test_arm()
-        elif tools.cross_building(self.settings) and self.settings.os == "Windows":
+        elif tools.cross_building(self.settings) and self.settings.os == "Windows" and self.settings.compiler != "Visual Studio":
```

Comment:
This seems a little odd, can you please why you changed it?

Reply:
When I ran `conan create`, the test phase failed for me, and my thinking was that it's probably due to the fact that I was using the VS compiler with the v141_xp toolset, which I guess will be considered cross compiling from win10.

Reply:
Hmm @uilianries would you have insight on this?